cnho_stables/assetlist.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "cnhostables",
```

Comment:
The chain name property here looks to have an acceptable value, but the name of the directory must match. Please change the directory name to: cnhostables (one word, no space or underscore)

Reply:
ok ,copy